tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -41,24 +41,28 @@
      * <p>
      * This method will check if the message is a
      * <ul>
-     * <li>{@link QueryMessage}, returning "{@code QueryMessage}",</li>
-     * <li>{@link CommandMessage}, returning "{@code CommandMessage}",</li>
-     * <li>{@link EventMessage}, returning "{@code EventMessage}",</li>
+     * <li>{@link QueryMessage}, returning  the queryName",</li>
```

Comment:
The javadoc isn't entirely in accordance to the code. The `queryName`/`commandName` are not necessarily returned in your adjustment, only when they _not_ equal the fully qualified class name of the `Message#getPayloadType`.

Reply:
corrected